nx-X11/programs/Xserver/hw/nxagent/Composite.c

```diff
@@ -32,7 +32,7 @@
 
 #define PANIC
 #define WARNING
-#undef  TEST
+#define  TEST
```

Comment:
Not supposed to be committed, undone in https://github.com/ArcticaProject/nx-libs/pull/125/commits/1dfd07b26a6c61bd191ef30dfc6630d415740d35.